atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -151,6 +155,7 @@ class CrashEditCoordsMap extends Component {
       isDragging,
     } = this.state;
     const geocoderAddress = this.props.mapGeocoderAddress;
+    const isDev = window.location.hostname === "localhost";
```

Comment:
You can set this value manually to `false` to cause the conditional style application later to be applied - useful for testing if you want to see it working.